CHANGELOG.md

```diff
@@ -5,113 +5,64 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Are there missing changes in the changelog? I mean the changes to Drydock backups after the migration to Drydock Tutor compatible version

Reply:
There are changes related to the [commits](https://github.com/eduNEXT/drydock-backups/commits/main/) given by [filter-repo](https://github.com/newren/git-filter-repo). I just follow these commits (ignoring some of them that are not relevant, like [this](https://github.com/eduNEXT/drydock-backups/commit/630aeef72b8c386879d76d0e07c20ffac71aa2cb) or [this](https://github.com/eduNEXT/drydock-backups/commit/55b63eb64a233926177c933a8f686648443d0398)). it seems are not missing changes, the last one did in drydock was [update docker image instructions](https://github.com/eduNEXT/drydock/tree/v15.5.1/drydock_backups)